Scripts/Runtime/Entities/BufferFromSingleEntity.cs

```diff
@@ -0,0 +1,38 @@
+using Unity.Entities;
+
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// A container that provides access to a <see cref="DynamicBuffer{T}" from a single entitiy.
+    /// </summary>
```

Comment:
Not sure if this is real or not but I'm not getting syntax highlighting on this showing it's a closing tag. All the other `</summary>` closing tags in this PR are highlighted properly. So maybe some random whitespace or special character involved that I'm not seeing? 

Or maybe just a bug with github.

Reply:
That is strange. I tried re-typing the line and previous line break. No diff.